sdk/batch/Azure.ResourceManager.Batch/src/Azure.ResourceManager.Batch.csproj

```diff
@@ -1,8 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.1.0-beta.1</Version>
-    <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.0.0</ApiCompatVersion>
+    <Version>1.1.0</Version>
```

Comment:
Please add back `<ApiCompatVersion>1.0.0</ApiCompatVersion>`.

Reply:
Yes, but we need it in this PR to run API compatibility check for `1.1.0` with the released `1.0.0`.

Reply:
Added it back.